src/legacy/js/functions/_t4ArticleEditor.js

```diff
@@ -167,14 +167,14 @@ function articleEditor(collectionId, data) {
     Florence.globalVars.pagePos = $(".workspace-edit").scrollTop();
 
     // tags
-    if ($("#selectTopic").val() && !$("#selectSubTopic").val()) {
-      sweetAlert("Cannot save this page", "A value is required for 'Subtopic' if a 'Topic' has been selected");
-      return
-    } 
-    else {
+    if ($("#selectTopic").val() && $("#selectSubTopic").val()) {
       data.description.primaryTopic = $("#selectTopic").val()[0]
       data.description.secondaryTopics = $("#selectSubTopic").val()
     }
+    else if ($("#selectTopic").val() && !$("#selectSubTopic").val()) {
+      sweetAlert("Cannot save this page", "A value is required for 'Subtopic' if a 'Topic' has been selected");
+      return
```

Comment:
It looks good but how would we feel with something like this instead? I find it would be more readable?
```
if ($("#selectTopic").val()) {
  if ($("#selectSubTopic").val()) {
        data.description.primaryTopic = $("#selectTopic").val()[0]
        data.description.secondaryTopics = $("#selectSubTopic").val()
  } else {
        sweetAlert("Cannot save this page", "A value is required for 'Subtopic' if a 'Topic' has been selected");
        return
  }
}
```